hello_world.cpp

```diff
@@ -0,0 +1,7 @@
+// Note: this file is just to mark this project as C++ related.
+#include <iostream>
+
+int main()
+{
+    std::cout << "Hello, world!\n";
```

Comment:
what the meaning of this pull request? Adding example code that can recommend for newbie?